tap_zendesk_chat/schemas/chats.json

```diff
@@ -41,7 +41,14 @@
     },
     "webpath": {
       "items": {
-        "$ref": "chat_webpath"
+        "anyOf": [
+          {
+            "$ref": "chat_webpath"
+          },
+          {
+            "$ref": "chat_webpath_array"
```

Comment:
Instead of creating 2 separate files, we can also put anyOf inside the chat_webpath file.

Reply:
Okay, will make this change.